tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -511,6 +532,10 @@ public function test_get_theme_items() {
 				),
 			),
 		);
-		$this->assertSameSetsWithIndex( $data, $expected );
+
+		wp_recursive_ksort( $data );
+		wp_recursive_ksort( $expected );
+
+		$this->assertSameSets( $data, $expected );
```

Comment:
Is it preferred sorting arrays before assertions, or can we use something like `assertEqualsCanonicalizing`?

Reply:
Here the deeply nested array of arrays are being recursively key sorted before the assertion. `assertSameSets()` repeats an index sort at the first dimension of the arrays. That's not necessary.

Instead of `assertSameSets()`, use `assertSame()`. 

I'm wondering though why this needed. Pulling down to explore.

Reply:
Hi, @hellofromtonya

Let me know if any follow-ups are needed for PHPUnits tests on this PR.